invoiced/objects.py

```diff
@@ -234,13 +324,28 @@ class Plan(CreateableObject, DeleteableObject, ListableObject,
            UpdateableObject):
     pass
 
-
 class Subscription(CreateableObject, DeleteableObject, ListableObject,
                    UpdateableObject):
 
     def cancel(self):
         return self.delete()
 
+    def preview(self, **params):
```

Comment:
Also think we should consider doing it this way here: https://github.com/Invoiced/invoiced-python/blob/master/invoiced/objects.py#L71

Reply:
This change has been made, as in the other libraries